src/pages/ServiceDetails/ServiceInfo/ServiceInfo.tsx

```diff
@@ -109,12 +109,14 @@ class ServiceInfo extends React.Component<ServiceId, ServiceInfoState> {
                           rightText={this.state.labels ? this.state.labels[key] : ''}
                         />
                       ))}
-                      <div className="progress-description">
-                        <strong>Type</strong> {this.state ? this.state.type : ''} <strong> Ip</strong>{' '}
-                        {this.state ? this.state.ip : ''}
+                      <div>
+                        <strong>Type</strong> {this.state ? this.state.type : ''}
```

Comment:
this.state is never supposed to be null or undefined afaik; what can be null or undefined, however, is `this.state.type`. So I would rather write `{this.state.type ? this.state.type : ''}`

Reply:
Actually `{undefined}` or `{null}` would just write nothing, so, I guess the null check is just useless